### PR TITLE
Minor documentation patches/fixes

### PR DESCRIPTION
**Pull request by abhro**

### CODE REVIEW

File: docs/src/documenter_to_vitepress_docs_example.md [hunk 1]
```diff
@@ -1,121 +1,118 @@
...
 Then the very first step here is to update the `make.jl` file to follow the DocumenterVitepress.jl format.
 
-1. Go the the `make.jl` file in your `docs` folder and do the following necessary changes to upgrade to DocumenterVitepress.jl :
+1. Go the the `make.jl` file in your `docs` folder and do the following necessary changes to upgrade to DocumenterVitepress.jl:
 
-:::tabs
+   :::tabs
 
-== For Documenter.jl
+   == For Documenter.jl
```
Comment (lazarusA) [inline docs/src/documenter_to_vitepress_docs_example.md:13]: Last time I wanna to change this as.

'From Documenter.jl'   ' to DocumenterVitepress jl' in the second tab. Could you do it here please 🙏.

File: docs/src/getting_started.md [hunk 2]
```diff
@@ -68,28 +68,28 @@ docs $
...
-However, to view your documentation live when developing locally, you will need to install `npm` and instantiate the 
+However, to view your documentation live when developing locally, you will need to install `npm` and instantiate the
 
 VitePress can be used on its own, or be installed into an existing project. In both cases, you can install it with:
 
 ::: code-group
 
 ```sh [npm]
-npm add -D vitepress
+$ npm add -D vitepress
```
Comment (lazarusA) [inline docs/src/getting_started.md:80]: I don't know how to feel about the dollar sign here. The idea is to just copy paste, having that symbol will cause some trouble to newcomers.

File: docs/src/documenter_to_vitepress_docs_example.md (hunk 1 above)
Comment (abhro) [reply docs/src/documenter_to_vitepress_docs_example.md:13]: I'm confused. Are you suggesting the following change?
```suggestion
   == From Documenter.jl to DocumenterVitePress.jl
```

Comment (lazarusA) [inline docs/src/documenter_to_vitepress_docs_example.md:13]:
```suggestion
   == From Documenter.jl
```

File: docs/src/documenter_to_vitepress_docs_example.md [hunk 3]
```diff
@@ -1,121 +1,118 @@
...
+       ],
+   )
+
+   deploydocs(;
+       repo = "github.com/jay-sanjay/Example.jl",
+       devbranch = "main",
+   )
+   ```
+
+   == For DocumenterVitepress.jl
```
Comment (lazarusA) [inline docs/src/documenter_to_vitepress_docs_example.md:46]:
```suggestion
   == to DocumenterVitepress.jl
```

File: docs/src/markdown-examples.md [hunk 4]
```diff
@@ -137,9 +137,9 @@ Note the combination with `[!code focus]`.
 ```julia
 using Gnuplot
 x = -2π:0.001:2π
-@gp x sin.(x) "w l t 'sin' lw 2 lc '#56B4E9'" "set gridss" # ‎ [!code error]
+@gp x sin.(x) "w l t 'sin' lw 2 lc '#56B4E9'" "set gridss" # [!code error]
 @gp :- xrange = (-2π - 0.3, 2π + 0.3) yrange = (-1.1,1.1)
-@gp :- x cos.(x) "w l t 'cos' lw 2 lc rgba '#E69F00'" # ‎[!code warning]
+@gp :- x cos.(x) "w l t 'cos' lw 2 lc rgba '#E69F00'" # [!code warning]
```
Comment (lazarusA) [inline docs/src/markdown-examples.md:142]: also, does this works for you locally? I kinda remember adding those invisible things for a reason.

Comment (lazarusA) [reply docs/src/markdown-examples.md]: what's is the effect? maybe nowadays they are not needed, I don't remember why 😢 I used them. Well... I will merge this now.

**Pull request merged**